File: app/src/main/java/com/jhost/eightball/MainActivity.java
package com.jhost.eightball;

import androidx.appcompat.app.AppCompatActivity;

import android.media.Image;
import android.os.Bundle;
import android.widget.Button;
import android.widget.ImageView;

import java.util.Random;

public class MainActivity extends AppCompatActivity {
    private final Random rng = new Random();

    private final int[] ballOptions = {
            R.drawable.ball1,
            R.drawable.ball2,
            R.drawable.ball3,
            R.drawable.ball4,
            R.drawable.ball5
    };

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);

        final Button askBtn = findViewById(R.id.askBtn);
        final ImageView ballImg = findViewById(R.id.ballImg);

        setRandomImg(ballImg);

        askBtn.setOnClickListener(event -> {
            setRandomImg(ballImg);
        });
    }

    private void setRandomImg(ImageView ballImg){
        ballImg.setImageResource(ballOptions[rng.nextInt(ballOptions.length)]);
    }
}

File: app/src/main/java/com/jhost/eightball/MainActivity.kt
package com.jhost.eightball

import android.os.Bundle
import android.widget.Button
import android.widget.ImageView
import androidx.appcompat.app.AppCompatActivity
import kotlin.random.Random

class MainActivity: AppCompatActivity(){

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)

        init()
    }

    private fun init(){
        val balls = intArrayOf(
            R.drawable.ball1,
            R.drawable.ball2,
            R.drawable.ball3,
            R.drawable.ball4,
            R.drawable.ball5
        )

        val askBtn: Button = findViewById(R.id.askBtn)
        val ballImg: ImageView = findViewById(R.id.ballImg)

        setRandomImage(ballImg, balls)

        askBtn.setOnClickListener {
            setRandomImage(ballImg, balls)
        }
    }

    private fun setRandomImage(imageView: ImageView, balls: IntArray){
        imageView.setImageResource(balls[Random.nextInt(balls.size)])
    }
}